README.md

```diff
@@ -49,12 +58,12 @@ You should then follow the [Cluster API Quick Start Guide](https://cluster-api.s
 
 If you do not change the generated `yaml` files, it will use defaults. You can look in the [templates/cluster-template.yaml](./templates/cluster-template.yaml) file for details.
 
-* `CPEM_VERSION`                 (defaults to `v3.6.2`)
-* `KUBE_VIP_VERSION`             (defaults to `v0.5.12`)
-* `NODE_OS`                      (defaults to `ubuntu_20_04`)
-* `POD_CIDR`                     (defaults to `192.168.0.0/16`)
-* `SERVICE_CIDR`                 (defaults to `172.26.0.0/16`)
-  
+- `CPEM_VERSION` (defaults to `v3.7.0`)
+- `KUBE_VIP_VERSION` (defaults to `v0.6.4`)
+- `NODE_OS` (defaults to `ubuntu_20_04`)
```

Comment:
noting that ubuntu_22_04 images had inconcistent package configurations - some with ifupdown others with netplan